src/schema/v2/me/__tests__/userInterest.test.ts

```diff
@@ -0,0 +1,100 @@
+import gql from "lib/gql"
+import { runAuthenticatedQuery } from "schema/v2/test/utils"
+import { ResolverContext } from "types/graphql"
+
+describe("me.userInterest", () => {
+  it("returns user interest", async () => {
+    const query = gql`
+      {
+        me {
+          userInterest(id: "user-interest-id") {
+            internalID
+            interest {
+              __typename
+              ... on Artist {
+                internalID
+              }
+            }
+          }
+        }
+      }
+    `
+    const meUserInterestLoader = jest.fn(async () => userInterestResponse)
+    const meLoader = jest.fn(async () => ({ id: "some-user-id" }))
+
+    const context: Partial<ResolverContext> = {
+      meUserInterestLoader,
+      meLoader,
+    }
+
+    const data = await runAuthenticatedQuery(query, context)
+
+    expect(meLoader).toHaveBeenCalled()
+    expect(meUserInterestLoader).toHaveBeenCalledWith("user-interest-id")
+
+    expect(data).toMatchInlineSnapshot(`
+      Object {
+        "me": Object {
+          "userInterest": Object {
+            "interest": Object {
+              "__typename": "Artist",
+              "internalID": "artist-id",
+            },
+            "internalID": "user-interest-id",
+          },
+        },
+      }
+    `)
+  })
+})
+
+const userInterestResponse = {
```

Comment:
**suggestion / non-blocking**: If the full response is needed for the test, thoughts on extracting this to a separate fixture file?

Reply:
Good idea 👍 I've extracted the fixture.